firefox-ios/Client/Frontend/Browser/EnhancedTrackingProtection/SlideoverPresentationController.swift

```diff
@@ -8,11 +8,12 @@ struct SlideOverUXConstants {
     static let ETPMenuCornerRadius: CGFloat = 8
 }
 
-class SlideOverPresentationController: UIPresentationController {
+class SlideOverPresentationController: UIPresentationController, FeatureFlaggable {
     let blurEffectView: UIVisualEffectView!
     var tapGestureRecognizer = UITapGestureRecognizer()
     var globalETPStatus: Bool
     weak var enhancedTrackingProtectionMenuDelegate: EnhancedTrackingProtectionMenuDelegate?
+    weak var legacyTrackingProtectionMenuDelegate: TrackingProtectionMenuDelegate?
```

Comment:
why are we adding additional implementation here for a legacy code?

Reply:
It has been renamed so we still want to keep the old delegate finish() method.

Reply:
okay, thank you for the explanation!